app/models/location.py

```diff
@@ -35,14 +44,14 @@ def create_zip_code(zip_code):
         return result
 
     @staticmethod
-    def generate_fake(count=10):
-        """Generate count fake ZIPCodes for testing."""
-        from faker import Faker
+    def generate_fake():
+        """Use a list of arbitrary but real zipcodes because the faker library
+            zipcodes are not verified & are not all valid"""
+        zip_codes = ['19104', '01810', '02420', '75205', '94305',
+                     '47906', '60521']
 
-        fake = Faker()
-
-        for i in range(count):
-            ZIPCode.create_zip_code(fake.zipcode())
+        for zip_code in zip_codes:
+            ZIPCode.create_zip_code(zip)
```

Comment:
change `zip` to `zip_code`. This code should have failed if you tried to run it, so please make sure you test code.